Portals/_default/Skins/Vanjaro/Base.ascx.cs

```diff
@@ -252,26 +264,6 @@ private void HandleUXM()
             }
         }
 
-        private void InjectAnalyticsScript()
-        {
-            if (HostController.Instance.GetBoolean("VJImprovementProgram", true))
-            {
-                AnalyticsManager.Post();
-
-                if (Analytics.RegisterScript)
-                {
-                    string PostEvent = Analytics.PostEvent();
-                    if (!string.IsNullOrEmpty(PostEvent))
-                    {
-                        WebForms.RegisterClientScriptBlock(Page, "GTagManager", "<script type='text/javascript' async src='https://www.googletagmanager.com/gtag/js?id=" + Analytics.MeasurementID + "'></script>", false);
-                        //WebForms.RegisterStartupScript(Page, "GTagManagerScript", "window.dataLayer = window.dataLayer || []; function gtag() { dataLayer.push(arguments); } gtag('js', new Date()); gtag('config', '" + Analytics.MeasurementID + "', { 'send_page_view': false }); " + PostEvent, true);
-                        string script = @"$(document).ready(function () {window.dataLayer = window.dataLayer || []; function gtag() { dataLayer.push(arguments); } gtag('js', new Date()); gtag('config', '" + Analytics.MeasurementID + "', { 'send_page_view': false }); " + PostEvent + "});";
-                        WebForms.RegisterStartupScript(Page, "AnalyticsPostEventScript", script, true);
-                    }
-                }
-            }
-        }
-
```

Comment:
@ashishpachauriap Why was this removed? @hshimanshusharma Please review this code

Reply:
It has been added.